src/rpc/blockchain.cpp

```diff
@@ -1879,6 +1879,9 @@ static UniValue getblockstats(const JSONRPCRequest& request)
     }
 
     const CBlock block = GetBlockChecked(pindex);
+    CBlockUndo blockUndo;
+    if (!UndoReadFromDisk(blockUndo, pindex))
```

Comment:
style-nit in commit ab4ed66cab35f5339921721cb85b57a1fd378f12:

Could you please add `{}` according to the developer notes and then squash the fixup commit into the first one?

https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

Reply:
addresses in https://github.com/bitcoin/bitcoin/commit/a565355d8f543cf158138f12130960863736bff3